CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [#8113](https://github.com/rubocop-hq/rubocop/pull/8113): Let `expect_offense` templates add variable-length whitespace with `_{foo}`. ([@eugeneius][])
 * [#8148](https://github.com/rubocop-hq/rubocop/pull/8148): Support autocorrection for `Style/MultilineTernaryOperator`. ([@koic][])
 * [#8151](https://github.com/rubocop-hq/rubocop/pull/8151): Support autocorrection for `Style/NestedTernaryOperator`. ([@koic][])
+* [#8160](https://github.com/rubocop-hq/rubocop/pull/8160): Add `rubocop-packaging` as a known extension in the docs. ([@utkarsh2102][])
```

Comment:
We typically don't mention doc updates in the changelog.

Reply:
Ah, apologies. Fixed! :smile: